src/module.js

```diff
@@ -770,6 +793,25 @@ export default class DailyIframe extends EventEmitter {
     return this._activeSpeakerMode;
   }
 
+  subscribeToTracksAutomatically() {
+    methodNotSupportedInReactNative();
+    return this._preloadCache.subscribeToTracksAutomatically;
+  }
+
+  setSubscribeToTracksAutomatically(enabled) {
+    methodNotSupportedInReactNative();
+    // only support this feature in call object mode
+    if (this._meetingState !== DAILY_STATE_NEW &&
+        !this._callObjectMode) {
+      console.error('track subscription is only supported in call object mode');
+      return;
+    }
+    this._preloadCache.subscribeToTracksAutomatically = enabled;
+    this.sendMessageToCallMachine({
```

Comment:
hm, what's the expected behavior when you toggle `setSubscribeToTracksAutomatically` mid-meeting? should a bunch of events be emitted saying tracks disappeared (if toggled off) or appeared (if toggled on)? or should only *changes* to tracks be emitted? usage of the word "subscribe" might imply the latter, even though that might not be intended...

if the intent is for all this new functionality (`setSubscribedTracks`, etc.) to control which participant tracks are *enabled* or *received*, it might be clearer if one of those words were used rather than "subscribed".

Reply:
either way, this will warrant plenty of clear documentation on the doc site! what might be helpful is a step-by-step walkthrough of a scenario in which the customer invokes `updateParticipants({setSubscribedTracks:{/*...*/}})` and `setSubscribeToTracksAutomatically(/*...*/)`, along what events are then emitted.

Reply:
oh, another possible framing of "subscribing automatically"—"defaulting to enabled for new tracks"

Reply:
I used "subscribed" because people porting from other video APIs will understand immediately what that means, I think. It's not necessarily the word I'd use if we were inventing this concept, but I'm inclined to use "subscribed" since we're not doing anything new, here (other than trying to pick good defaults and be easier to use than other APIs).

Also, "enabled" is overloaded because that's a low-level track property that doesn't quite mean the same thing as what we're doing here (sadly).

Reply:
👍

Reply:
after finishing this review and the `pluot-core` one, I got used to the "subscribe" terminology and have come to rather like it